src/content/docs/ko/guides/cms/directus.mdx

```diff
@@ -12,6 +12,7 @@ i18nReady: false
 ## 공식 자료
 
 - Directus는 [Astro 블로그 템플릿 예시](https://github.com/directus/examples/tree/main/astro)를 제공합니다.
+- [Astro와 함께 Directus 시작하기](https://docs.directus.io/blog/getting-started-directus-astro.html).
```

Comment:
This new resource should be overwriting the old one, since it doesn't exist anymore.